docs/premium-features/custom-types-and-actions.md

```diff
@@ -132,14 +198,22 @@ This is useful when you want to temporarily disable certain endpoints without co
 
 To enable or disable an Action, simply toggle the switch next to the Action's name in the APIs tab.
 
+:::caution
+Disabling an Action will cause Amplication not to generate the code for that Action.
+:::
+
 ![Enable and Disable Actions](./assets/custom-types-and-actions/all-actions.png)
 
 ### Edit and Delete Action
 
-Similar to DTOs, you can edit, delete, move, and duplicate Actions within the APIs tab. When editing an Action, you can modify its HTTP method, path, input DTO, output DTO, and other settings as needed. Remember to save your changes when you're done.
+To edit an Action, click on the Action within its associated Module in the APIs tab. You can modify the Action's properties, such as the HTTP method, path, input DTO, output DTO, and other settings as needed. Remember to save your changes when you're done.
+
+To delete an Action, click on the delete button next to the Action's name. Be cautious when deleting Actions, as it may impact other parts of your application that depend on them.
```

Comment:
Let's mention here that Amplication Actions (built-in, or such) cannot be deleted